keyboards/acheron/keebspcb/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "KeebsPCB", 
+    "url": "http://gondolindrim.github.io/AcheronDocs/keebs/intro.html", 
+    "maintainer": "Gondolindrim", 
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```

I just realized this keyboard is `LAYOUT_60_ansi_tsangan`.

```suggestion
        "LAYOUT_60_ansi_tsangan": {
```